docs/processes/release/tasks/release-build.md

```diff
@@ -24,185 +24,40 @@ License & terms of use: http://www.unicode.org/copyright.html
-   again while running the ant release target.
-4. Run `$ ant releaseFinal` . This target generates release files for ICU site,
-   as well as files for Maven Central Repository, then upload the files to OSSRH
-   staging repository. You should see build log as below at the end.
-   ```
-      [deploy] Transferring 37K from icu4j-releases
-      [deploy] Uploaded 37K
-      [deploy] Uploading: com/ibm/icu/icu4j-localespi/61.1/icu4j-localespi-61.1-sources.jar.asc to repository icu4j-releases at https://oss.sonatype.org/service/local/staging/deploy/maven2
-      [deploy] Transferring 0K from icu4j-releases
-      [deploy] Uploaded 0K
-      [deploy] Uploading: com/ibm/icu/icu4j-localespi/61.1/icu4j-localespi-61.1-javadoc.jar to repository icu4j-releases at https://oss.sonatype.org/service/local/staging/deploy/maven2
-      [deploy] Transferring 22K from icu4j-releases
-      [deploy] Uploaded 22K
-      [deploy] Uploading: com/ibm/icu/icu4j-localespi/61.1/icu4j-localespi-61.1-javadoc.jar.asc to repository icu4j-releases at https://oss.sonatype.org/service/local/staging/deploy/maven2
-      [deploy] Transferring 0K from icu4j-releases
-      [deploy] Uploaded 0K
-
-   BUILD SUCCESSFUL
-   Total time: 6 minutes 19 seconds
-   ```
-   After the successful target execution, copy files generated under `release`
-   directory and upload them to ICU site.
-5. Log on to [OSSRH Nexus Repository Manager](https://oss.sonatype.org/). Select
-   "Staging Repositories" under "Build Promotion" on the left navigation pane,
-   then type "com.ibm.icu" on the search field on the upper right corner. You
-   should find a repository entry corresponding to the one just uploaded. Select
-   the entry, and sanity check content. Once you confirm the contents, click
-   "Close" on the menu. If the repository is closed successfully, you can
-   "Release" after that. The release operation will promote the staging
-   repository to production. It usually takes 20 to 30 minutes.
-
-![image](NexusRepoMan.png)
+as well as the ICU GitHub releases.
+
+This is all done by a GitHub Action,
+see [publish/release.md](publish/release.md),
+the _"Release - ICU4J publish to Maven Central"_ section.
+
+The previous step stages the Maven artifacts to Sonatype, but does
+not automatically push them to Maven Central. \
+It can be enabled, but we chose no to, so that a human can do a last sanity check. \
+So someone must login to Sonatype, check that everything looks fine, and approve. \
+To do that log on to the [Sonatype Central Portal -- Namespaces](https://central.sonatype.com/publishing/namespaces). \
+Select the "Deployments" tab. \
+Check the files staged there, compare to a previous public release in Maven Central
+(for example [ICU4J 77.1](https://repo1.maven.org/maven2/com/ibm/icu/icu4j/77.1/)
+and [ICU4J Charset 77.1](https://repo1.maven.org/maven2/com/ibm/icu/icu4j-charset/77.1/)) \
+Once you confirm the contents, approve it. \
```

Comment:
If these are steps for doing an approval of a deployment, then put them in an ordered list.

If the approval step is something that has a technical step to it (clicking a putting, updating a setting in some UI), then describe that with words. The screenshot might help, but words are still important.